CMakeLists.txt

```diff
@@ -151,8 +152,8 @@ list(APPEND LPEG_DEPENDENCIES "luajit")
 ExternalProject_Add(
   lpeg
   DEPENDS ${LPEG_DEPENDENCIES}
-  URL "https://s3.amazonaws.com/download.draios.com/dependencies/lpeg-1.0.0.tar.gz"
-  URL_HASH "SHA256=10190ae758a22a16415429a9eb70344cf29cbda738a6962a9f94a732340abf8e"
+  URL "http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.2.tar.gz"
```

Comment:
Here are the changelogs I could find.
I also looked at the code and can't see any API change.
```

* Changes from version 1.0.1 to 1.0.2
  ---------------------------------
  + some bugs fixed

* Changes from version 0.12 to 1.0
  ---------------------------------
  + group "names" can be any Lua value
  + some bugs fixed
  + other small improvements
```

Reply:
I had the same troubles here.

Version 1 has been removed and substituted by 1.0.2 (that only contains some bug fixes).